packages/server/src/tests/utilities/structures.ts

```diff
@@ -241,6 +243,21 @@ export function collectAutomation(tableId?: string): Automation {
   return automation as Automation
 }
 
+export function basicAutomationLog(automationId: string): AutomationResults {
```

Comment:
Naming here is somewhat confusing. Is it an `AutomationLog` or `AutomationResults`?

Reply:
Good point, yes this returns an automation result, mis-typed the function name!